services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/PathHelper.groovy

```diff
@@ -68,8 +67,11 @@ class PathHelper {
             throw new Exception("Unable to make some path more preferable because both paths use same ISLs")
         }
         log.debug "ISL to avoid: $islToAvoid"
-        northbound.updateLinkProps([islUtils.toLinkProps(islToAvoid, ["cost": NOT_PREFERABLE_COST]),
-                                    islUtils.toLinkProps(islToAvoid.reversed, ["cost": NOT_PREFERABLE_COST])])
+        def maxAvailableCost = morePreferableIsls.findAll { it != islToAvoid }.collect {
+            northbound.getLink(it)
+        }.cost.max()
+        northbound.updateLinkProps([islUtils.toLinkProps(islToAvoid,
+                ["cost": (maxAvailableCost + NOT_PREFERABLE_COST).toString()])])
```

Comment:
I literally see no sense in the `maxAvailableCost` variable and it's impact. The preferred path always depends on the sum of all isl costs on that path, having the single maximum cost gives you nothing, since two paths can have different amount of isls.
How does setting `maxAvailableCost + NOT_PREFERABLE_COST` instead of `NOT_PREFERABLE_COST` improves the algo?

Reply:
In a nutshell I thought it's more safe then just set `NOT_PREFERABLE_COST`.

This algo was rewritten and now I compare total cost of both paths and then make a decision about creating/updating linkProps
```
  def totalCostOfMorePrefPath = morePreferableIsls.collect {
            northbound.getLink(it)
        }.cost.sum()
        def totalCostOfLessPrefPath = getInvolvedIsls(lessPreferablePath).collect {
            northbound.getLink(it)
        }.cost.sum()
        if(totalCostOfLessPrefPath < totalCostOfMorePrefPath) {
            northbound.updateLinkProps([islUtils.toLinkProps(islToAvoid,
                    ["cost": (totalCostOfMorePrefPath).toString()])])
        }
```